src/sql/platform/capabilities/test/common/testCapabilitiesService.ts

```diff
@@ -33,6 +33,7 @@ export class TestCapabilitiesService implements ICapabilitiesService {
 				defaultValue: undefined!,
 				isIdentity: true,
 				isRequired: true,
+				showOnConnectionDialog: false,
```

Comment:
once you make this optional property, you shouldn't need to add it if the value is false.